src/main/python/KeyboardToNetworkTables.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import time
+from typing import Callable, TypeVar
+
+from pynput import keyboard
+
+from NetworkTableManager import NetworkTableInstance, NetworkTable, NetworkTableClient
+
+SpecialKey = TypeVar("SpecialKey")
+ASCIIKey = TypeVar("ASCIIKey")
+UndefinedKey = TypeVar("UndefinedKey")
+
+KEYBOARD_EVENT_CHECKING_COOLDOWN_SECONDS = 0.01
```

Comment:
It can be higher. Worst case the thread that is going to be closed will be killed.

Reply:
fixed by still opened because when I do things according to my own cr i don't know if I so shit. 

So if it's good change resolve this, otherwise revert it.

Reply:
Is 0.05 high enough

Reply:
why not make it .1? Though it doesn't really matter since it runs on an external computer anyway.